index.d.ts

```diff
@@ -206,6 +208,19 @@ export interface Options extends Omit<RequestInit, 'headers'> {
 	*/
 	json?: unknown;
 
+	/**
+	User-defined JSON-parsing function.
+
+	Useful e.g in these cases:
+
+	1. Parse JSON via [bourne](https://github.com/hapijs/bourne) to protect from prototype pollution.
+	2. Parse JSON with [reviver](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse).
+
+	@default JSON.parse()
```

Comment:
Missing the example from the readme. Readme and index.d.ts should be in sync.

Reply:
fixed